functions.sh

```diff
@@ -1960,17 +1970,19 @@ make_swraid() {
       fi
 
       # create raid array
-      if echo "$line" | grep "$LASTDRIVE" >/dev/null ; then
+      if grep -q "$LASTDRIVE" <<<"$line" ; then
```

Comment:
The same applies here: `<<< "$line"`

Reply:
If feasible, variables should be enclosed by curly `"${brackets}"`

Reply:
see above.